<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>App Turismo</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(151, 151, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnMenu">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../Sistemas en Python/elecciones/assets/iconos/menu.png</normaloff>../../../../Sistemas en Python/elecciones/assets/iconos/menu.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QFrame" name="frame_menu_lateral">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>200</width>
      <height>631</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_titulo">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Bienvenido al backend de la App</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_usuario">
       <property name="placeholderText">
        <string>Usuario</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_password">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Contraseña</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnLoginAceptar">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Iniciar Sesión</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCerrarSesion">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Cerrar Sesión</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_estado_login">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLabel" name="label_foto_usuario">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string>Foto Usuario</string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnGestionCargas">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Gestión Registros</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnBackendDeploy">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>24</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 7pt;
padding-left: 20px;
background-color: #f2f2f2;
color: #333;</string>
       </property>
       <property name="text">
        <string>🔧 Deploy Backend </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnBuildDeploy">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>24</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 7pt;
padding-left: 20px;
background-color: #f2f2f2;
color: #333;</string>
       </property>
       <property name="text">
        <string>🚀 Build &amp; Deploy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnConfiguracion">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>24</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 7pt;
padding-left: 20px;
background-color: #f2f2f2;
color: #333;</string>
       </property>
       <property name="text">
        <string>Configuración General</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnUsuarios">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>24</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 7pt;
padding-left: 20px;
background-color: #f2f2f2;
color: #333;


</string>
       </property>
       <property name="text">
        <string>Usuarios</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRegionesZonas">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>24</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 7pt;
padding-left: 20px;
background-color: #f2f2f2;
color: #333;

</string>
       </property>
       <property name="text">
        <string>Regiones/Zonas</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSecciones">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>24</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 7pt;
padding-left: 20px;
background-color: #f2f2f2;
color: #333;

</string>
       </property>
       <property name="text">
        <string>Secciones</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSubSecciones">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>24</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 7pt;
padding-left: 20px;
background-color: #f2f2f2;
color: #333;</string>
       </property>
       <property name="text">
        <string>Sub-Secciones</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSalir">
       <property name="text">
        <string>Salir del Sistema</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_imagen_central">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>150</y>
      <width>450</width>
      <height>450</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>btnMenu</tabstop>
  <tabstop>lineEdit_usuario</tabstop>
  <tabstop>lineEdit_password</tabstop>
  <tabstop>btnLoginAceptar</tabstop>
  <tabstop>btnGestionCargas</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
